source/guides/basics/custom-items.md

```diff
@@ -1,26 +1,198 @@
 Fancier Sticks: Making Custom Items (PLACEHOLDER ONLY)
 -----------------------------------
 
-**TODO: Write-up that introduces item script containers, and the general ideas of making custom items, mentioning recipes, name/lore, mechanisms, item flags, etc.**
-
-### Placeholder
+```eval_rst
+.. contents:: Table of Contents
+    :local:
+```
 
-Until this page is written, you can view the [old tutorial video here](https://one.denizenscript.com/denizen/vids/Custom%20Items).
+### What Are Custom Items
 
-### Sample Script
+Custom items are a way to store a various set of properties onto an item to create a new unique item in the game. Denizen provides the capabilities to save these unique items in item scripts so that they can be reused or uniquely referenced to in other scripts. These were briefly mentioned in [the mechanisms guide](/guides/basics/mechanisms) but this page will go into more detail.
 
-Here's a quick sample of a modern item script.
+### The Basic Structure
 
+To start off with, this is the basic structure of an item script:
```

Comment:
This isn't the structure of an item script, this is an example of an item script.